pkg/tests/tests.go

```diff
@@ -108,3 +117,35 @@ func resourceReviewbody() []byte {
 	}
 	return buff.Bytes()
 }
+
+func clusterAuthorizationsBody() []byte {
+	buff := &bytes.Buffer{}
+	reservedResource := acctmgmtv1.NewReservedResource()
+
+	clusterAuthReq, err := acctmgmtv1.NewClusterAuthorizationRequest().
+		AccountUsername(helpers.AccountUsername).
+		AvailabilityZone("???").
```

Comment:
"single" or "multi"

Reply:
"single", based on the MW00530 sku granted to your org

Reply:
but if OCP, neither value really matters as all OCP maps to 0 cost.  See row 25:

https://docs.google.com/spreadsheets/d/1HGvQnahZCxb_zYH2kSnnTFsxy9MM49vywd-P0X_ISLA/edit#gid=968570191